lib/recaptcha/helpers.rb

```diff
@@ -271,6 +271,8 @@ def self.recaptcha_v3_default_callback_name(action)
     private_class_method def self.default_callback(options = {})
       nonce = options[:nonce]
       nonce_attr = " nonce='#{nonce}'" if nonce
+      selector_tag = options[:selector]
+      selector_attr = options[:selector] ? "[data-selector='#{selector_tag}']" : '.g-recaptcha'
```

Comment:
I'd have expected something like this:
```suggestion
      selector_attr = options[:selector] || '.g-recaptcha'
```

Reply:
I don't think that would work, would it? For `data-selector="my-selector"`, `options[:selector]` would return `my-selector`. As far as I know, to select by a `data-` attribute you need the whole thing?

Reply:
I could inline it though:
``` ruby
selector_attr = options[:selector] ? "[data-selector='#{options[:selector]}']" : '.g-recaptcha'
```

Reply:
let the users pass a query-selector, that seems to be the most obvious api to me ... they can still send in `[data-selector='whatever']` ?

Reply:
Not a 100% sure if I understand. My intention was to allow:
``` ruby
<%= invisible_recaptcha_tags selector: "my-selector" %>
```
It then gets transformed to `data-selector='my-selector'` by the `components` method. Hence the code in `.default_callback` is using `data-selector=`.

So are you suggesting to do something like this?
``` ruby
<%= invisible_recaptcha_tags selector: ".my-class", class: ".my-class" %>
# or
<%= invisible_recaptcha_tags selector: "[data-selector='my-selector']", "data-selector": ".my-selector" %>
```

Reply:
ah I think I get it now, the selector attr goes into the tag anyway
how about using `id` for that, it's supposed to be unique and would make a fast selector too ?

Reply:
Ah yeah using the id will probably be better! I'll make the change in this PR in the next few days.